pages/price-feeds/getting-started.mdx

```diff
@@ -0,0 +1,15 @@
+## Getting Started
+
+Integrating Pyth price feeds into your application is quick and easy.
+
+### Build 🧑‍💻
+
+- Jump to our quick [tutorial](./create-your-first-pyth-app/) to get started with Pyth price feeds in your application.
+- Consult our [guides](./use-real-time-data/) and selected the appropriate chain for your application.
+
+**NOTE**: Pyth price feeds are permissionless and do not require a subscription or API key -- developers can complete the whole process in a few minutes.
```

Comment:
(Imo this would be good content for the first paragraph of the section. You want to tell people "hey it's easy to integrate, just pick the relevant one of these things for you")